docs/tutorials/gs_continuous_space.ipynb

```diff
@@ -0,0 +1,1210 @@
+{
```

Comment:
<p>can we put the 1/2 in v(x) as well? it's a bit strange to leave it outside I find </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/netket/netket/pull/1574/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
it was to show that you can do things like float * nk.operator, but indeed it does not really fit the tex hamiltonian